tests/ha/single_pod_restart/test_server_pod_restart_api.py

```diff
@@ -160,8 +162,7 @@ def teardown_method(self):
         LOGGER.info("STARTED: Teardown Operations.")
         if self.s3_clean:
             LOGGER.info("Cleanup: Cleaning created s3 accounts and buckets.")
-            resp = self.ha_obj.delete_s3_acc_buckets_objects(self.s3_clean, is_ver=self.is_ver,
-                                                             v_etag=self.version_etag)
+            resp = self.ha_obj.delete_s3_acc_buckets_objects(self.s3_clean)
```

Comment:
please try running all tests affected here

Reply:
Ran 1-1 IO test of data and server pod, and it is working fine.

Reply:
Have you tried running tests which don't change or have multiple versions?

Reply:
Yes. One test with delete IOs, one on versioning from server and data pod tests